packages/console/src/pages/SignInExperience/components/Skeleton/index.module.scss

```diff
@@ -2,145 +2,146 @@
-      margin-top: _.unit(4);
-      @include _.shimmering-animation;
-    }
+  .cardTitle {
+    flex-shrink: 0;
+  }
 
-    .tabs {
-      width: 498px;
-      height: 25px;
-      margin: _.unit(6) 0 _.unit(15.5);
-      @include _.shimmering-animation;
-    }
+  .tabBar {
+    width: 100%;
+    height: 28px;
+    @include _.shimmering-animation;
+  }
 
-    .field {
-      width: 100%;
-      height: 44px;
-      margin-top: _.unit(6);
-      @include _.shimmering-animation;
-    }
+  .content {
+    width: 100%;
+    display: flex;
 
-    .footer {
-      display: flex;
-      width: 100%;
-      justify-content: right;
+    .setup {
+      flex: 1;
+      margin-right: _.unit(3);
 
-      .button {
-        width: 194px;
-        height: 44px;
-        @include _.shimmering-animation;
+      > :not(:first-child) {
+        margin-top: _.unit(3);
       }
-    }
-  }
 
-  .preview {
-    width: 480px;
-    overflow: hidden;
+      .card {
+        background-color: var(--color-layer-1);
+        border-radius: 16px;
+        padding: _.unit(6);
 
-    .header {
-      display: flex;
-      width: 100%;
-      height: 103px;
-      padding: _.unit(6);
+        >:not(:first-child) {
+          margin-top: _.unit(6);
+        }
 
-      .info {
         .title {
-          width: 113px;
-          height: 20px;
           @include _.shimmering-animation;
+          width: 80px;
+          height: 16px;
         }
 
-        .subtitle {
-          width: 177px;
-          height: 16px;
-          margin-top: _.unit(4);
+        .field {
           @include _.shimmering-animation;
+          width: 100%;
+          height: 44px;
         }
       }
-
-      .button {
-        width: 104px;
-        height: 30px;
-        margin-left: _.unit(2.5);
-        @include _.shimmering-animation;
-      }
     }
 
-    .content {
-      background: var(--color-surface-variant);
+    .preview {
+      flex: 1;
 
-      .mobile {
+      .header {
         display: flex;
-        flex-direction: column;
-        align-items: center;
-        width: 300px;
-        height: 649px;
-        background: var(--color-surface);
-        margin: _.unit(10) auto 0;
+        width: 100%;
+        height: 103px;
         padding: _.unit(6);
-        border-radius: 16px;
-        transform: scale(0.65);
-        transform-origin: top;
-
-        .logo {
-          width: 64px;
-          height: 64px;
-          margin-top: _.unit(16);
-          @include _.shimmering-animation;
-        }
-
-        .slogan {
-          width: 177px;
-          height: 16px;
-          margin: _.unit(3) 0 _.unit(10);
-          @include _.shimmering-animation;
-        }
-
-        .field {
-          width: 100%;
-          height: 36px;
-          margin-top: _.unit(3);
-          @include _.shimmering-animation;
+        background-color: var(--color-layer-1);
+        border-radius: 16px 16px 0 0;
+
+        .info {
+          .title {
+            width: 113px;
```

Comment:
ditto and for all the rest hard coded width number bellow

Reply:
I think it's not a big deal, we don't change its size, just adjust the code position.

Reply:
https://www.figma.com/file/iroUuWbeZ9fqcqXNwqYZai/%F0%9F%92%BB-%5BAC%5D-Admin-Console-V1-Master?node-id=6581%3A231829&t=9Qdo9hQWpBvVXlZE-4